actionmailer/lib/action_mailer/base.rb

```diff
@@ -392,6 +392,7 @@ module ActionMailer
   #     of an OpenSSL verify constant (<tt>'none'</tt>, <tt>'peer'</tt>, <tt>'client_once'</tt>,
   #     <tt>'fail_if_no_peer_cert'</tt>) or directly the constant (<tt>OpenSSL::SSL::VERIFY_NONE</tt>,
   #     <tt>OpenSSL::SSL::VERIFY_PEER</tt>, ...).
+  #	<tt>:ssl/:tls</tt> Enables Action Mailer to use SMTP/TLS (SMTPS: SMTP over direct TLS connection)
```

Comment:
The alignment issue seems to be problem of GitHub's diff inside the PR. Went to my local file (and the full file in GitHub) and the alignment is correct. However, I'll rewrite it.